requirements/docs.txt

```diff
@@ -406,19 +406,6 @@ starlette==0.41.3 \
     --hash=sha256:0e4ab3d16522a255be6b28260b938eae2482f98ce5cc934cb08dce8dc3ba5835 \
     --hash=sha256:44cedb2b7c77a9de33a8b74b2b90e9f50d11fcf25d8270ea525ad71a25374ff7
     # via sphinx-autobuild
-tornado==6.4.1 \
```

Comment:
FYI it was dropped here:  _(in 2024.04.13)_

https://github.com/sphinx-doc/sphinx-autobuild/commit/ca7eebeb98d7987f0a6dbef159dafa9f7e5c7f3d

(from `livereload`/`tornado` to `uvicorn`/`websockets` and `starlette`…)